test/examples/fail_dont_repeat_yourself.erl

```diff
@@ -10,13 +10,16 @@ repeated_complexity_5(X, Y) ->
     W = Y ++ [ok],
     Z ++ W ++ [ok].
 
+-spec repeated_complexity_10(any(), any()) -> fun((...) -> any()).
 repeated_complexity_10(X, Y) ->
-    case X of
-        Y -> io:format("Y");
-        _ -> <<"ok">>
-    end,
+    Z = case X of
+            Y -> io:format("Y");
+            _ -> <<"ok">>
+        end,
 
-    case Y of
-        X -> io:format("Y");
-        _ -> <<"ok">>
-    end.
+    W = case Z of
+            X -> io:format("Y");
```

Comment:
This file is just for testing purposes, so that duplicated structures are detected by the `dont_repeat_yourself` rule. The code doesn't necessarily have to be functional.